testing/tests/DevExpress.ui.widgets.dataGrid/editing.tests.js

```diff
@@ -8791,6 +8791,42 @@ QUnit.module('Refresh modes', {
         // assert
         assert.ok(event.isDefaultPrevented(), 'default is prevented');
     });
+
+    ['Full', 'Reshape', 'Repaint'].forEach(refreshMode => {
+        QUnit.test(`${refreshMode} - selectedRowKeys should be updated after deleting a row (T896582)`, function(assert) {
```

Comment:
just a suggestion - why not
```
// T896582
['Full', 'Reshape', 'Repaint'].forEach(refreshMode => {
        QUnit.test(`selectedRowKeys should be updated after deleting a row (editMode=${refreshMode})`, function(assert) {
```

Reply:
I did it intentionally to display test names with a capital latter.